octoprint_mrbeam/static/css/tab_workingarea.css

```diff
@@ -759,4 +762,5 @@ body.ctrlKey #translateHandle {
   stroke-dasharray: none;
   stroke-dashoffset: 0;
   stroke-opacity: 1;
+  pointer-events: none;
```

Comment:
This is the fix for #1178 (same commit) - This is not an issue usually with classical merges, but can be problematic with our squash merge practice.

Reply:
agree